appveyor.yml

```diff
@@ -52,6 +52,10 @@ build_script:
 
 environment:
   NOT_CRAN: true
+  # See SPARK-27848. Currently installing some dependent packagess causes
+  # "(converted from warning) unable to identify current timezone 'C':" for an unknown reason.
+  # This environment variable works around to test SparkR against a higher version.
+  R_REMOTES_NO_ERRORS_FROM_WARNINGS: true
```

Comment:
Maybe, another workaround might be possible, but the current one also looks more robust.
```
Setting _R_CHECK_FORCE_SUGGESTS_=false while R_REMOTES_NO_ERRORS_FROM_WARNINGS is unset 
will also avoid stopping the installation for error messages.
This is done because a warning is generated during installation 
when not all Suggested packages are not available.
```

Reply:
Hmhm, I haven't tried this before but seems like this only applies when we install `sparkr`. The error happened during manual `testthat` installation (at `praise` https://github.com/r-lib/testthat/blob/master/DESCRIPTION#L24)

Maybe it could work too for the reason I missed for now but I believe `R_REMOTES_NO_ERRORS_FROM_WARNINGS` is fine too.